task/build-paketo-ubi-builder/0.1/build-paketo-ubi-builder.yaml

```diff
@@ -0,0 +1,527 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: build-paketo-ubi-builder
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: image-build, buildpack, paketo, konflux
+  labels:
+    app.kubernetes.io/version: "0.1"
+    build.appstudio.redhat.com/build_type: pack
+spec:
+  description: |-
+    Build-paketo-ubi-builder task builds a ubi image of a paketo builder project using as input the builder.toml file.
+    The task also produces the SBOM which is signed and added to the image.
+  params:
+    - name: BUILD_ARGS
+      description: the arguments to be passed to the pack command to build the image
+      type: array
+      default: []
+    - name: BUILDER_IMAGE
+      description: The image packaging the paketo tools and to be used to build
+      type: string
+      default: "quay.io/redhat-user-workloads/konflux-build-pipeli-tenant/paketo-container:ea8ddb8818bb4a55546927e7674b0362dabd6342"
```

Comment:
IIRC we cannot allow builder image to be specified as param, it must be hardcoded in task to provide valid provenance, other build tasks have been updated to disallow such parameter.

Reply:
also image should be officially released in konflux-ci quay namespace, we don't allow images from user-workloads. This image should be properly released via releases to konflux-ci

Reply:
> also image should be officially released in konflux-ci quay namespace, we don't allow images from user-workloads. This image should be properly released via releases to konflux-ci

What should we do to release it part of konflux-ci ?

Reply:
I removed the default value in the meantime. See: a328f429

Reply:
you have to configure your konflux instance with proper release plan

Reply:
> we cannot allow builder image to be specified as param

and what we do about this part?

Reply:
> and what we do about this part?

What do you mean ?

Reply:
we cannot allow users to specify their own images, it must be hardcoded in the task to provide correct provenance

Reply:
Ok. So asap as this image will be released by konflux, then we will set the image ref to be used.